src/Comment.php

```diff
@@ -204,6 +204,14 @@ public static function parse(string $json, bool $assoc = false, int $depth = 512
 
     public static function parseFromFile(string $file, bool $assoc = false, int $depth = 512, int $options = 0)
     {
+        if (!file_exists($file)) {
+            throw new \RuntimeException(sprintf('File %s does not exist', $file));
+        }
+
+        if (!is_file($file)) {
+            throw new \RuntimeException(sprintf('%s is not a file', $file));
```

Comment:
maybe we can use a better exception msg here?

Reply:
also , maybe better to use InvalidArgumentException?

Reply:
Do you have a suggestion for a better message? I think "[filename] does not exist or is not a file" (or maybe "not a *regular* file"?) is pretty much what you would need to know, right?

I'm not sure about the InvalidArgumentException, since that's seems more for things where the wrong *type* is provided. So for example if an object was given instead of a filename.
But like I said in the issue: I'm not sure myself.

Reply:
yes please. i would say:
- only is_file check
- msg: [filename] does not exist or is not a file
- exception: InvalidArgumentException

that's all. 😎